### DISCUSSION TITLE
Keyboard shortcut to switch data view and properties

### DISCUSSION THREAD

**Discussion by Elliska** [dbeaver/dbeaver#30086] [Q&A]
I searched through entire keyboard shortcuts list and could not find any help. I am trying to achieve switching between tabs by a keyboard.

Generally I take F4 (view table) from navigation tree and then have three tabs at my disposal: Properties, Data and ER Diagram. How can I switch between those three tabs? 

Or at least how it would be called so I can bind it with my own shortcuts.

![image](https://github.com/dbeaver/dbeaver/assets/108549406/9ef5f61c-55f0-4afb-8100-ef1bae3d744f)

**Comment by msi-end** [comment]
**Check Application Documentation:** Sometimes, keyboard shortcuts might not be explicitly listed in the application's interface but could be documented in the user manual or help documentation. Check if there's any documentation available for your application online or within the application itself.
**Customize Keyboard Shortcuts:** Some applications allow users to customize keyboard shortcuts. Look for any options in your application's settings or preferences menu that allow you to define or modify keyboard shortcuts. You might be able to assign shortcuts to tab navigation commands if this feature is supported.
**Use System-Level Shortcuts:** If your application doesn't provide built-in shortcuts for tab navigation, you can try using system-level shortcuts or third-party tools to achieve similar functionality. For example, on Windows, you can use Alt + Tab to switch between open windows, including tabs within an application. On macOS, you can use Command + Tab to switch between applications and Command + Tilde (`) to switch between windows of the same application.
**Scripting or Automation:** Depending on the capabilities of your application, you might be able to automate tab navigation using scripting or automation tools. This approach typically requires some programming knowledge and might not be feasible for all applications.

**Comment by LonwoLonwo** [comment]
Hello @Elliska 

Please try to use **Alt+PageDown**
